File: src/main/java/de/fhopf/akka/PageRetriever.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package de.fhopf.akka;

/**
 *
 * @author Florian Hopf, http://www.florian-hopf.de
 */
public interface PageRetriever {

    PageContent fetchPageContent(String url);
    
}

File: src/main/scala/de/fhopf/akka/PageRetriever.scala
package de.fhopf.akka

/**
 * @author Florian Hopf, http://www.florian-hopf.de
 * @author Alfredo Serafini, http://www.seralf.it
 */
abstract class PageRetriever {
  def fetchPageContent(url: String): PageContent
}
